src/main/java/projectLibraryPartOne/service/LibraryService.java

```diff
@@ -0,0 +1,87 @@
+package projectLibraryPartOne.service;
+
+import projectLibraryPartOne.entity.Book;
+
+import java.util.ArrayList;
+
+public class LibraryService {
```

Comment:
It is better create interface LibraryService and then it's implementation with realization all methods

Reply:
corrected